mysql/init.sql

```diff
@@ -98,6 +98,16 @@ CREATE TABLE IF NOT EXISTS unizin_metadata (
     pvalue VARCHAR(100)
 );
 
+-- CREATE TABLE IF NOT EXISTS user_default_selection (
+--      id int NOT NULL AUTO_INCREMENT,
+--      course_id CHAR(255),
+--      user_id CHAR(255),
+--      default_view_type CHAR(225),
+--      default_view_value CHAR(225),
+--      PRIMARY KEY (course_id, user_id, default_view_type)
+-- );
```

Comment:
I commented out the create table so that if we want to use Django model to create this table.